frontend/src/components/employees/index.tsx

```diff
@@ -136,9 +141,65 @@ const addButton = (
   </div>
 );
 
+const deleteModalBody = (
+  staff: StaffData,
+  closeModalFunction: () => void,
+  deleteCallback: CallbackFunction
+) => (
+  <div className={employeesStyle.modalDeleteContent}>
+    <div className={employeesStyle.modalDeleteHeader}>
+      <h4>Delete</h4>
+      <button
+        type="button"
+        aria-label="closeButton"
+        className={employeesStyle.xButton}
+        onClick={closeModalFunction}>
+        <Clear />
+      </button>
+    </div>
+    <hr />
+    <span>
+      Are you sure you want to delete {staff.firstName} {staff.surname}?
+    </span>
+    <div className={employeesStyle.modalDeleteButtons}>
+      <Button variant="contained" onClick={closeModalFunction}>
+        Cancel
+      </Button>
+      <Button
+        variant="contained"
+        color="secondary"
+        onClick={deleteCallback(staff)}>
+        DELETE
+      </Button>
+    </div>
+  </div>
+);
+
```

Comment:
is it worth taking the modal out of this file and make it's own thing? I feel we are making this file to cluttered.

Reply:
done